R/get_plan_doc_info.R

```diff
@@ -0,0 +1,36 @@
+#' Get planning doc info
+#'
+#' Get the specific information for the upcoming workshop so it can be used in the planning doc
+#'
+#' @param ws_dat information about the workshop to be inserted into the planning document
+#' @param meta_fld the URL of the meta folder
+#'
+#' @return planning doc information to be used in rendering the doc
+#'
+#' @export
+#'
+get_plan_doc_info <- function(ws_dat, meta_fld) {
```

Comment:
As with `get_debrief_doc_info.R` I do not see the `meta_fld` argument in use in the body of the function. Are you sure it is needed?